pkg/services/stateroot/service.go

```diff
@@ -118,6 +118,14 @@ func (s *service) OnPayload(ep *payload.Extensible) error {
 			s.log.Error("can't add SV-signed state root", zap.Error(err))
 			return nil
 		}
+		s.srMtx.Lock()
+		ir, ok := s.incompleteRoots[sr.Index]
+		if ok {
+			ir.Lock()
+			ir.isSent = true
+			ir.Unlock()
+		}
+		s.srMtx.Unlock()
```

Comment:
Can we move this unlock before the `if`?

Reply:
Fixed.